openc3/lib/openc3/api/tlm_api.rb

```diff
@@ -118,7 +118,8 @@ def set_tlm(*args, type: :CONVERTED, scope: $openc3_scope, token: $openc3_token)
     # @param type [Symbol] Telemetry type, :RAW, :CONVERTED (default), :FORMATTED, or :WITH_UNITS
     def inject_tlm(target_name, packet_name, item_hash = nil, type: :CONVERTED, scope: $openc3_scope, token: $openc3_token)
       authorize(permission: 'tlm_set', target_name: target_name, packet_name: packet_name, scope: scope, token: token)
-      unless CvtModel::VALUE_TYPES.include?(type.intern)
+      type = type.to_s.intern
+      unless CvtModel::VALUE_TYPES.include?(type)
```

Comment:
Why this change? You can call `intern` on an existing symbol ... it's a noop.

Reply:
handles nil and strings coming in (which is what was happening in inject_tlm)